.travis.yml

```diff
@@ -195,6 +196,7 @@ before_script: |
 script: |
   echo "Calling pytest with the following arguments: $PYTEST_ADDOPTS"
   python -mpytest
+  tox -e pytz
```

Comment:
Is that doing two full test suites, one with and one without pytz?  If yes, that's way too much; if no, what is this doing?

In general I agree with @jklymak that we probably don't want the test suite to be based on tox.

Reply:
No, `tox -e pytz` only runs the `pytz` environment, which runs [`pytest -m pytz`](https://github.com/matplotlib/matplotlib/pull/11360/files#diff-b91f3d5bd63fcd17221b267e851608e8R25), and thus only runs tests decorated with `pytest.mark.pytz`.

Reply:
try adding a `set -e` to the script section?  or combining the two lines via `&`.  I think the issue is that the return value from the script section is the return of the last thing called.

Reply:
Yes, I came to this conclusion as well, though the main problem with this is that if the first one fails the second one won't be run at all.

Ideally the main test suite would *also* be run with `tox`, so that you can do:

```
tox -e py,pytz
```

Which would run both environments and have a single return code. If the project doesn't want to use tox as the main test runner, we can use the `-e` or `pytest ... && tox -e pytz` solution as a "for now" and leave it to a later PR to refactor the test suite as preferred.